test-data/unit/check-incremental.test

```diff
@@ -3467,6 +3467,68 @@ tmp/main.py:2: error: Expression has type "Any"
 [out2]
 tmp/main.py:2: error: Expression has type "Any"
 
+[case testAttrsIncrementalSubclassingCached]
+from a import A
+import attr
+@attr.s
+class B(A):
+    pass
+reveal_type(B)
+[file a.py]
+import attr
+@attr.s
+class A:
+    x: int = attr.ib()
+
+[out1]
+main:6: error: Revealed type is 'def (x: builtins.int) -> __main__.B'
+[out2]
+main:6: error: Revealed type is 'def (x: builtins.int) -> __main__.B'
```

Comment:
Please add more tests for incremental mode. Here are some ideas:
* Check situations with more attributes
* Checks that access to attributes after deserialization results in a correct type and "kind" (i.e. being frozen etc.)
* Check that signatures of deserialized dunder methods are correct.
* Check situations where base attr class in module `a.py` is not changed in the second run, but its subclass in module `b.py` is changed so that this produces a type error.
* Also vice-versa (error to no error)
* Add test with more than two files
* Add test with more than two incremental runs (for example: no error -> error -> no error in a subclass)

Here are two additional notes about incremental tests:
* If a file results in errors, it is never actually deserialized from cache, this _includes_ `reveal_type`. So although you can keep current tests (to check serialization, which happens always), please avoid `reveal_type` in future incremental tests, instead use an assignment to check that the type is as expected.
* To add changes between runs, use this pattern:
```
[file a.py]
import b
...
[file a.py.2]
import b
...
[file b.py]
...
```
Important note: you don't need to add `.2`, (and `.3` in one test) to all files, only to the files that are actually changed. It is better to actually _not_ do this (to avoid `touch`ing unchanged file, and therefore considering its cache stale).

Don't add to many tests, just one case for each item I listed above.